docs/BasicCommunication/OnSystemTimeReady/index.md

```diff
@@ -0,0 +1,37 @@
+## OnSystemTimeReady
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Notifies SDL that the HMI is ready to provide accurate system time
+
+!!! must
+
+The HMI must send a BC.OnSystemTimeReady notification to SDL when a new ignition cycle occurs in order to notify SDL that the HMI is once again ready to provide accurate system time.
+!!!
+
+### Notification
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
```

Comment:
Looks like the table is normally omitted and `This RPC has no additional parameter requirements` is included here if the RPC has no parameters

Reply:
@jacobkeeler, thank you
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/b2c0188108ecf9f1875672583eec5f7f52e7ecaf